courses/free-courses-hi.md

```diff
@@ -200,6 +200,7 @@
 * [Data Structure And Algorithms Course | DSA Tutorial in Hindi](https://www.youtube.com/playlist?list=PLmGElG-9wxc9Us6IK6Qy-KHlG_F3IS6Q9) - ScoreShala
 * [Data Structure Programs \| Hindi](https://www.youtube.com/playlist?list=PLsFNQxKNzefJU-Sj__mljvrmJHZVKWbEm) - Saurabh Shukla
 * [Data Structures and Algorithms Course in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ahIappRPN0MCAgtOu3lQjQi) - CodeWithHarry
+* [Data Structures Using Java for Interview Preperations](https://youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ&si=y2djZoQ_GKx-KzBP) - Kunal Kushwaha
```

Comment:
Don't invent titles! Use the title given by the resource.
Remove the trackin (`si`) parameter

Reply:
The Changes has been made. Thanks for the Suggestions.